contracts/FractalRegistry.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IFractalRegistry.sol";
+
+/// @notice A contract for registering Fractal DAO name strings
+/// @notice These names are non-unique, and should not be used as the identifer of a DAO
```

Comment:
These comments only apply to the `updateDAOName` function, so should be moved down there

Reply:
and maybe a not about what this contract does here now